server/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
+
+require('dotenv').config();
+
+const app = express();
+const config = require('../webpack.config.js');
```

Comment:
All imports should be done at the top of the file

Reply:
Refactor server file to have imports at the top of the file.